tests/unit/cn/cn_open_test.c

```diff
@@ -87,11 +88,19 @@ setup_mocks(void)
     mapi_inject_list_set(inject_list);
 }
 
+struct cn_kvdb *cn_kvdb;
+
 static int
 pre(struct mtf_test_info *lcl_ti)
 {
+    merr_t err;
+
     setup_mocks();
 
+    err = cn_kvdb_create(4, 4, &cn_kvdb);
+    if (err)
```

Comment:
Same abort() comment

Reply:
Probably returning errno is worse than calling abort from a debugging standpoint, lemme see if we can make pre return a merr_t...  Although fixing the pre return code would probably require touching a lot of files, so then probably just return errno for now...

Reply:
I think the test frameworks sucks hard in this regard. When the collection pre funcs fail, there is no indication at all if I remember correctly, just silent failure.